### PR TITLE
Updated DICOM dictionary to 2019a. Connected to #724

### LINKED ISSUE #724: Support DICOM Standard 2019a
See [Release Notes](http://dicom.nema.org/medical/dicom/current/output/pdf/releasenotes_2019a.pdf).

### PR DESCRIPTION
**Pull request by anders9ustafsson**
Fixes #724.

#### Checklist
- [x] The pull request branch is in sync with latest commit on the *fo-dicom/development* branch
- [ ] I have updated API documentation
- [ ] I have included unit tests
- [x] I have updated the change log
- [x] I am listed in the CONTRIBUTORS file

#### Changes proposed in this pull request:
- Updated UIDs, tags and anonymizer profiles to DICOM dictionary 2019a.
- Added new anonymization profile *Retain Institution Identification Option*.
- Changed names on retired transfer syntax fields.

### CODE REVIEW

**Comment by anders9ustafsson** [comment]
Apparently there is an issue with the DICOM dictionary .gz file right now. Will investigate further and post a correction commit as soon as possible.

File: DICOM/DicomAnonymizer.cs [hunk 1]
```diff
@@ -38,17 +38,19 @@ public enum SecurityProfileOptions : short
 
             RetainDeviceIdent = 8,
 
-            RetainPatientChars = 16,
+            RetainInstitutionIdent = 16,
```
Comment (gofal) [inline DICOM/DicomAnonymizer.cs]: wouldn't it be better to assign the new RetainInstitutionIdent the new value 1024 and keep the numbers of the  already existing enums? I know from gitter discussions that some store the combined interger value as configuration to apply anonymization level.

Comment (anders9ustafsson) [reply DICOM/DicomAnonymizer.cs]: Problem is that the flag also represents the order in the anonymizer table in the DICOM standard, and the *Retain Institution Identification Option* has been inserted in the middle of the table. So this change was necessary to maintain correct functionality.

Comment (gofal) [reply DICOM/DicomAnonymizer.cs]: I see. Then we should not forget to put this as breaking change into the release notes at least.

**Pull request merged**